node/src/accountant/test_utils.rs

```diff
@@ -1644,3 +1678,131 @@ impl ScanSchedulers {
         }
     }
 }
+
+#[derive(Default)]
+pub struct PayablePaymentsAgentMock {
```

Comment:
You mentioned wanting to change some of the names below, using the word `set` instead of `set_up` or `conclude`